### DISCUSSION TITLE
Release date of milestone version that works well with PMD 7

### DISCUSSION THREAD

**Discussion by ZhaoxiongChen** [jborgers/PMD-jPinpoint-rules#372] [Q&A] [answered]
Hi, I was wondering if you got any release date for a milestone version that is compatible with PMD 7?

**Comment by dylan-tao** [comment, marked as answer]
> Hi, I was wondering if you got any release date for a milestone version that is compatible with PMD 7?

The author of the current rules is already doing this, I believe we will meet in the near future, the following link is the current progress: https://github.com/jborgers/PMD-jPinpoint-rules/issues/309
